pom.xml

```diff
@@ -162,5 +167,89 @@
         </plugins>
       </build>
     </profile>
+
+      <!-- Activate this profile to generate licenses deliverables -->
+      <!-- Note that we cannot use id "profile" as it's already defined by booster parent -->
```

Comment:
Should that be "licenses" and not "profile"?

Reply:
Ah, yes of course. It should be _"licenses" profile id_.